powerauth-push-server/src/main/java/com/wultra/push/controller/rest/PushCampaignController.java

```diff
@@ -91,12 +89,14 @@ public PushCampaignController(AppCredentialsRepository appCredentialsRepository,
     public ObjectResponse<CreateCampaignResponse> createCampaign(@RequestBody ObjectRequest<CreateCampaignRequest> request) throws PushServerException {
         final CreateCampaignRequest requestObject = request.getRequestObject();
         if (requestObject == null) {
+            logger.error("action: createCampaign, state: failed, error: Request object must not be empty");
```

Comment:
This branch could be improved in the future in a separate PR to use bean validation too, instead of an explicit check.

Reply:
@zcgandcomp Do you want to solve this now or later on?